File: Python/PRAK504-2210817220009-Azka Amalia.py
def reverse(num):
    n = 0
    while(int(num)):
        rem = int(num)%10
        n = n*10+rem
        num = int(num)/10
    return n
A,B = map(int,input().split())
A = reverse(A)
B = reverse(B)
C = A+B
print(reverse(C))

File: C/PRAK504-2210817220009-Azka Amalia.c
#include<stdio.h>
int reverse(int num){
    int n=0, rem;
   while(num){
        rem=num%10;
        n=n*10+rem;
        num=num/10;}
        return n;
}
int main(){
    int A, B;
    scanf("%d%d",&A,&B);
    A=reverse(A);
    B=reverse(B);
    int C = A+B;
    printf("%d",reverse(C));
}
